jwst/conftest.py

```diff
@@ -73,3 +74,30 @@ def jail(request, tmpdir_factory):
     os.chdir(str(newpath))
     yield newpath
     os.chdir(old_dir)
+
+@pytest.mark.trylast
+def pytest_configure(config):
+    terminal_reporter = config.pluginmanager.getplugin('terminalreporter')
+    config.pluginmanager.register(TestDescriptionPlugin(terminal_reporter), 'testdescription')
+
+class TestDescriptionPlugin:
+
```

Comment:
One last request before we merge this - can you add a docstring for this class to indicate what it does?  And maybe add a couple comment strings down below as well?  Mostly for our future selves to know what this code is doing here.